common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -47,6 +47,8 @@ public static void init(String modVersion) {
 
         ModelLoader.init();
         FeatureRegistry.init();
+
+        System.setProperty("java.awt.headless", "false");
```

Comment:
What is this used for? Can you add a comment explaining?

Reply:
Okay,
> you can't set the contents of the clipboard if you're running headless

A comment explaining would be nice, it is not that obvious.

Reply:
Yeah my bad, good point

Reply:
@P0keDev Does this really work? Afaik the java.awt.headless property cannot be changed after Java has started...

Reply:
I can confirm that, at least in a dev environment, commenting out that line will cause the game to crash. Maybe it works because Java knows somehow that we're not actually headless?